.github/workflows/dco.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+# yamllint disable rule:line-length
+
+name: "DCO check"
```

Comment:
Can we change the `name` of this workflow here from "DCO check" to "Developer Certificate of Origin" so that it's obvious what it's doing at a glance, even to people who have never typed "what is dco?" into google?

Reply:
@grenade renamed as expected.